automation/capitanodoc/utils.ts

```diff
@@ -15,41 +15,9 @@
  * limitations under the License.
  */
 
-import type { OptionDefinition } from 'capitano';
-import * as ent from 'ent';
 import * as fs from 'fs';
 import * as readline from 'readline';
 
-export function getOptionPrefix(signature: string) {
-	if (signature.length > 1) {
-		return '--';
-	} else {
-		return '-';
-	}
-}
-
-export function getOptionSignature(signature: string) {
-	return `${getOptionPrefix(signature)}${signature}`;
-}
-
-export function parseCapitanoOption(option: OptionDefinition): string {
```

Comment:
These 3 functions were never used. I suppose capitano is the thing we used/created before we moved to oclif (https://github.com/balena-io/capitano). Capitano is not even a dependency of the cli anymore and all of these are not used anywhere